ceph/mds.go

```diff
@@ -361,50 +386,56 @@ func (m *MDSCollector) collectMDSSlowOps() {
 			return
 		}
 
+		var metricMap sync.Map
```

Comment:
Not sure I see the point of this being a `sync.Map` since this is all single threaded.

Reply:
Should provide sync semantics and easy to split this into Go routines in case we want to spin one per MDS (possibly in next iteration).